ethsigner/signer/file-based/src/main/java/tech/pegasys/ethsigner/signer/filebased/FileBasedSubCommand.java

```diff
@@ -27,9 +27,8 @@
 @Command(
     name = FileBasedSubCommand.COMMAND_NAME,
     description =
-        "This command ensures that received transactions are signed by a key stored in an encrypted file.",
-    mixinStandardHelpOptions = true,
-    helpCommand = true)
+        "This command ensures transactions are signed by a key stored in an encrypted file.",
+    mixinStandardHelpOptions = true)
```

Comment:
Does this sub command also need the helpCommand = true value like the HashicorpSubCommand?

Reply:
think this needs to be removed all together! :thumbsup: